src/main/java/com/solvd/sql/interfaces/IProductPromotionDao.java

```diff
@@ -0,0 +1,7 @@
+package com.solvd.sql.interfaces;
+
+import com.solvd.sql.model.ProductPromotion;
+
+public interface IProductPromotionDao extends IBaseDAO<ProductPromotion> {
+    ProductPromotion getByProductName(String productName);
+}
```

Comment:
```suggestion
public interface IProductPromotionDao extends IBaseDAO<ProductPromotion> {

    ProductPromotion getByProductName(String productName);
    
}
```
just to make it more readable